README.md

```diff
@@ -214,7 +214,14 @@ const bot = new Telegraf(token);
 bot.on("text", ctx => ctx.reply("Hello"));
 
 // Start webhook via launch method (preferred)
-bot.launch({ webhook: { domain: webhookDomain, port: port } });
+bot.launch({ 
+  webhook: {
+    domain: webhookDomain, // https://example.com
+    port: port, // 443
+    hookPath: webhookPath, // Application path on server
+    secretToken: randomAlphaNumericString // Optional but recommmended for more security. Also have to provide while setting the webhook url. see https://core.telegram.org/bots/api#setwebhook
```

Comment:
`setWebhook` is not needed when using `bot.launch()`.  It's called internally.

Maybe also recommend how to securely generate the token. On Node, this could be: `crypto.randomBytes(64).toString('hex')`

Reply:
Specified token generation in the comment, please review it.